docs/examples/CreatableInputOnly.tsx

```diff
@@ -17,60 +16,33 @@ const createOption = (label: string) => ({
   value: label,
 });
 
-interface State {
-  readonly inputValue: string;
-  readonly value: readonly Option[];
-}
+export default () => {
+  const [inputValue, setInputValue] = React.useState('');
+  const [value, setValue] = React.useState<readonly Option[]>([]);
 
-export default class CreatableInputOnly extends Component<{}, State> {
-  state: State = {
-    inputValue: '',
-    value: [],
-  };
-  handleChange = (
-    value: OnChangeValue<Option, true>,
-    actionMeta: ActionMeta<Option>
-  ) => {
-    console.group('Value Changed');
-    console.log(value);
-    console.log(`action: ${actionMeta.action}`);
-    console.groupEnd();
```

Comment:
I've removed the console logs to simplify the example. I don't think they were necessary for the purpose of this example.